scss/civicrm/administer/permissions/_access.scss

```diff
@@ -11,14 +10,35 @@
   }
 }
 
-&.page-civicrm-admin-synchuser {
-
+.page-civicrm-admin-synchuser {
   .crm-form-block {
-    background: none;
-    box-shadow: none;
-
     .help {
       display: none;
     }
   }
 }
+
+.page-civicrm-acl,
+.page-civicrm-admin-options-acl-role {
+  #crm-main-content-wrapper {
+    #wizard-steps {
+      @include civicrm-panel;
+    } 
+  }
+}
+
+.page-civicrm-acl {
+  .dataTables_wrapper {
+    @include civicrm-table;
+  }
+}
+
+.page-civicrm-acl-entityrole {
+  @import "../common/table/table";
```

Comment:
same as before, this should be included only once in a common main file. we don't want a bunch of imports scattered and hidden everywhere